src/main/java/net/sf/jabref/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -452,6 +449,7 @@ private String getResults(URL source) throws IOException {
                 sb.append((char) buffer[i]);
             }
         }
+        in.close();
```

Comment:
OK! I'll change that. I'd say that there are quite a few cases where this could be changed in JabRef. The ones I went for were only the warnings of missing `.close()`.

Again, just to understand, is it a difference when one open a resource which will "always" open? That is, not a file or network request, such as Scanner (which I guess has fewer potential problem sources)?